assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -11,7 +11,7 @@
 
 ### CHE SERVER
 # Folder where Che will store internal data objects
-che.database=${che.home}/storage
+che.database=${che.home}
```

Comment:
why it's not anymore in a subfolder ?

Reply:
It was like so in Che instances, that were running through CLI for a long while because of this che.env.erb:
https://github.com/eclipse/che/blob/56866159228bf4ba1d3b7fc8d2206c60b129579b/dockerfiles/init/modules/che/templates/che.env.erb#L36
So, the values from the che.properties or entrypoint.sh, which would point to path with "storage" directory had no effect, until recently (which was the issue cause)

I decided to consolidate everything by old path, though if you think extra "storage" directory is better, it can be changed to that

Reply:
I would say that it looks odd to have a "root" directory as path for che database so yes using a subfolder would be better IMHO


https://github.com/eclipse/che/blob/93b974d1e0ec7836635f3d7da196b7651da8ad1b/dockerfiles/che/entrypoint.sh#L233

Reply:
@benoitf Am I understand you correctly. You want to change db file location.
Add extra path "storage" to what we have in Che5?

Reply:
yes. From what I see there is a 'db' subfolder added
It's not directly stored inside the che.home folder